crates/cext/src/dag.rs

```diff
@@ -583,6 +585,167 @@ pub unsafe extern "C" fn qk_dag_apply_gate(
     }
 }
 
+/// @ingroup QkDag
+/// Apply a measure to a DAG.
+///
+/// @param dag The circuit to apply to.
+/// @param qubit The qubit index to measure.
+/// @param clbit The clbit index to store the result in.
+/// @param front Whether to apply the measure at the start of the circuit. Usually `false`.
+///
+/// @return The node index of the created instruction.
+///
+/// # Safety
+///
+/// Behavior is undefined if `dag` is not an aligned, non-null pointer to a valid :c:struct:`QkDag`,
```

Comment:
When adding references within a docstring, avoid using `:c:..:` these don't get linked, using a pair of ``  works better and links correctly. This goes for all examples in which it was used.
```suggestion
/// Behavior is undefined if `dag` is not an aligned, non-null pointer to a valid ``QkDag``,
```

Reply:
If this is the case, then I think we've got Doxygen configured incorrectly.  It probably should be the case that we _don't_ attempt to parse Markdown out of it and pass it through verbatim for Sphinx to interpret, so we can use the full power of all cross-references.